test/units/testsuite-58.sh

```diff
@@ -2,6 +2,11 @@
 set -eux
 set -o pipefail
 
+if ! which systemd-repart &>/dev/null; then
```

Comment:
this will probably make shellcheck angry, it usually asks to use `command -v`

Reply:
Ack.